src/content/developers/docs/nodes-and-clients/index.md

```diff
@@ -19,18 +19,51 @@ If you're new to the topic of nodes, we recommend first checking out our user-fr
 
 ![Coupled execution and consensus clients](./eth1eth2client.png)
 Simplified diagram of a coupled execution and consensus client.
 
+### Client diversity {#client-diversity}
+
+Both [execution clients](/developers/docs/nodes-and-clients/#consensus-clients) and [consensus clients](/developers/docs/nodes-and-clients/#execution-clients) exist in a variety of programming languages developed by different teams.
+
+Multiple client implementations can make the network stronger by reducing its dependency on a single codebase. The ideal goal is to achieve diversity without any client dominating the network, thereby eliminating a potential single point of failure. 
+The variety of languages also invites a broader developer community and allows them to create integrations in their preferred language.
+
+Learn more about [client diversity](/developers/docs/client-diversity).
+
+What these implementations have in common is they all follow a single specification. Specifications dictate how the Ethereum network and blockchain functions. Every technical detail is defined and specifications can be found as:
+
+- Originally, the [Ethereum Yellow Paper](https://ethereum.github.io/yellowpaper/paper.pdf)
+- [Execution specs](https://github.com/ethereum/execution-specs/)
+- [Consensus specs](https://github.com/ethereum/consensus-specs)
+- [EIPs](https://eips.ethereum.org/) implemented in various [network upgrades](/history/)
+
+### Tracking nodes in the network {#network-overview}
+
+Multiple trackers offer a real-time overview of nodes in the Ethereum network. Note that due to the nature of decentralized networks, these crawlers can only provide a limited view of the network and might report different results.
+
+- [Map of nodes](https://etherscan.io/nodetracker) by Etherscan
+- [Ethernodes](https://ethernodes.org/) by Bitfly
+- [Ethereum Node Crawler](https://crawler.ethereum.org/)
+- [Nodewatch](https://www.nodewatch.io/) by Chainsafe, crawling consensus nodes
+
 ## Node types {#node-types}
 
-If you want to [run your own node](/developers/docs/nodes-and-clients/run-a-node/), you should understand that there are different types of node that consume data differently. In fact, clients can run 3 different types of node - light, full and archive. There are also options of different sync strategies which enables faster synchronization time. Synchronization refers to how quickly it can get the most up-to-date information on Ethereum's state.
+If you want to [run your own node](/developers/docs/nodes-and-clients/run-a-node/), you should understand that there are different types of node that consume data differently. In fact, clients can run three different types of node—light, full and archive. There are also options of different sync strategies which enable faster synchronization time. Synchronization refers to how quickly it can get the most up-to-date information on Ethereum's state.
```

Comment:
```suggestion
If you want to [run your own node](/developers/docs/nodes-and-clients/run-a-node/), you should understand that there are different types of node that consume data differently. In fact, clients can run three different types of nodes—light, full and archive. There are also options of different sync strategies which enable faster synchronization time. Synchronization refers to how quickly it can get the most up-to-date information on Ethereum's state.
```
Should be plural